OvmfPkg/IoMmuDxe/AmdSevIoMmu.c

```diff
@@ -936,6 +1392,19 @@ InstallIoMmuProtocol (
     goto CloseExitBootEvent;
   }
 
+  //
+  // Currently only Tdx guest support Reserved shared memory for DMA operation.
+  //
+  if (TdIsEnabled ()) {
+    mReservedSharedMemSupported = TRUE;

```

Comment:
Only enable this for TD path? Is there any improvement for non-TD case?

Reply:
It benefits SEV as well. SEV can be enabled easily. But let AMD do the enabling. Because I don't have the SEV environment to test the change.